src/ZoneServer/Skills/Handlers/Archer/ObliqueShot.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Melia.Shared.L10N;
+using Melia.Shared.Tos.Const;
+using Melia.Zone.Network;
+using Melia.Zone.Skills.Combat;
+using Melia.Zone.Skills.Handlers.Base;
+using Melia.Zone.Skills.SplashAreas;
+using Melia.Zone.World.Actors;
+using Melia.Zone.World.Actors.CombatEntities.Components;
+using static Melia.Zone.Skills.SkillUseFunctions;
+
+namespace Melia.Zone.Skills.Handlers.Archer
+{
+	/// <summary>
+	/// Handles the Archer skill Oblique Shot.
+	/// </summary>
+	[SkillHandler(SkillId.Archer_ObliqueShot)]
+	public class ObliqueFire : ITargetSkillHandler
+	{
+		/// <summary>
+		/// Handles the skill, shoot missile at enemy that spreads to another target.
+		/// </summary>
+		public void Handle(Skill skill, ICombatEntity caster, ICombatEntity designatedTarget)
+		{
+			if (!caster.TrySpendSp(skill))
+			{
+				caster.ServerMessage(Localization.Get("Not enough SP."));
+				return;
+			}
+
+			skill.IncreaseOverheat();
+			caster.TurnTowards(designatedTarget);
+
+			if (designatedTarget == null)
+			{
+				return;
+			}
+
+			if (!caster.Position.InRange2D(designatedTarget.Position, skill.Data.MaxRange))
+			{
+				caster.ServerMessage(Localization.Get("Too far away."));
+				return;
+			}
+
+			this.Attack(skill, caster, designatedTarget);
+		}
+
+		/// <summary>
+		/// Executes the actual attack.
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		/// <param name="designatedTarget"></param>
+		private async void Attack(Skill skill, ICombatEntity caster, ICombatEntity designatedTarget)
+		{
+			var damageDelay = TimeSpan.Zero;
```

Comment:
I logged a delay of 45, does it work correctly with 0?

Reply:
it seems to do so, 45ms is almost imperceptible.

Reply:
That might be, but do keep in mind that we're still in the process of figuring out how all of this works, and there is a big difference between "no delay" and "some delay", because "some delay" must come from somewhere. Any information we might get, like what kinds of values to expect, will help us figure out more down the line.